src/styles/typography.scss

```diff
@@ -20,6 +20,11 @@
   }
 }
 
+.text-red {
+  font-size: 16px;
+  color: red;
+}
+
```

Comment:
Das war ein Missverständnis. text-red ist schon in Vuetify integriert. Zusammen mit text-body sind dann beide Stile abgedeckt. Siehe Template

Reply:
Oh danke! done.